--- a/BUILD
+++ b/BUILD
@@ -2,7 +2,7 @@ load("@aspect_bazel_lib//lib:jq.bzl", "jq")
 load("//:macros.bzl", "json_data")
 load("//:versions.bzl", "VERSIONS")
 
-exports_files(["jq.sh"])
+exports_files(["jq.sh", "versions.bzl"])
 
 json_data(
     name = "deps",
@@ -13,6 +13,7 @@ json_data(
 jq(
     name = "dependency_versions",
     srcs = [":deps"],
+    visibility = ["//visibility:public"],
     out = "dependency_shas.json",
     filter = """
     with_entries(select(.value | objects and .type == "github_archive") | .value |= {repo, sha256, url, version})
