--- emscripten_toolchain/toolchain.bzl.orig
+++ emscripten_toolchain/toolchain.bzl
@@ -14,6 +14,10 @@
     "with_feature_set",
     _flag_set = "flag_set",
 )
+load(
+    "@rules_cc//cc/toolchains:cc_toolchain_config_info.bzl",
+    "CcToolchainConfigInfo",
+)
 load("@bazel_tools//tools/build_defs/cc:action_names.bzl", "ACTION_NAMES")
 
 def flag_set(flags = None, features = None, not_features = None, **kwargs):
