--- remote_emscripten_repository.bzl.orig
+++ remote_emscripten_repository.bzl
@@ -2,6 +2,7 @@
 load(":emscripten_build_file.bzl", "EMSCRIPTEN_BUILD_FILE_CONTENT_TEMPLATE")
 load(":revisions.bzl", "EMSCRIPTEN_TAGS")
 load("//emscripten_toolchain:toolchain.bzl", "emscripten_cc_toolchain_config_rule")
+load("@rules_cc//cc:defs.bzl", "cc_toolchain", "cc_toolchain_suite")
 
 def remote_emscripten_repository(
     name,
@@ -117,7 +118,7 @@
         }),
     )
 
-    native.cc_toolchain(
+    cc_toolchain(
         name = cc_wasm_name,
         all_files = all_files_target,
         ar_files = ar_files_target,
@@ -139,7 +140,7 @@
         toolchain_type = "@bazel_tools//tools/cpp:toolchain_type",
     )
 
-    native.cc_toolchain_suite(
+    cc_toolchain_suite(
         name = "everything-" + name,
         toolchains = {
             "wasm": cc_wasm_target,
