--- emscripten_toolchain/BUILD.bazel.orig
+++ emscripten_toolchain/BUILD.bazel
@@ -1,6 +1,7 @@
 load("//:emscripten_deps.bzl", "emscripten_repo_name")
 load("//:remote_emscripten_repository.bzl", "create_toolchains", "emscripten_toolchain_name")
 load("@rules_python//python:py_binary.bzl", "py_binary")
+load("@rules_cc//cc:defs.bzl", "cc_library")
 
 package(default_visibility = ["//visibility:public"])
 
