--- protoc-gen-validate-1.3.0-original/bazel/protobuf.bzl	2025-12-11 06:18:06.448563483 +0000
+++ protoc-gen-validate-1.3.0/bazel/protobuf.bzl	2025-12-11 06:18:15.322575895 +0000
@@ -1,6 +1,8 @@
 load("@bazel_skylib//rules:common_settings.bzl", "BuildSettingInfo")
-load("@bazel_tools//tools/jdk:toolchain_utils.bzl", "find_java_runtime_toolchain", "find_java_toolchain")
+load("@bazel_tools//tools/jdk:toolchain_utils.bzl", "find_java_toolchain")
 load("@com_google_protobuf//bazel/common:proto_info.bzl", "ProtoInfo")
+load("@rules_java//java/common:java_common.bzl", "java_common")
+load("@rules_java//java/common:java_info.bzl", "JavaInfo")
 
 # Borrowed from https://github.com/grpc/grpc-java/blob/v1.28.0/java_grpc_library.bzl#L59
 # "repository" here is for Bazel builds that span multiple WORKSPACES.
